tests/src/comparators/test_byGivenOrder.cpp

```diff
@@ -0,0 +1,54 @@
+/* test_byGivenOrder.cpp
+Copyright (c) 2022 by Michael Zahniser
```

Comment:
You might still want to change the copyright.

Reply:
@jostephd were you going to do this?

Reply:
I'm happy with that comment as is... does it need to be changed?

Reply:
From the 👍 I wasn't sure of your intent. Since you say it's fine as-is, fine by me.